### PR TITLE
Draft ietf lpwan schc over lorawan 09 - Following Eric's review

### PR DESCRIPTION
**Pull request by Oliv4945**
Answered Eric V. comments, mostly to add details and correct few typos
@ivajloip can you please review ?
@jca-klk do you also want to review it ?

### CODE REVIEW

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 1]
```diff
@@ -99,33 +94,40 @@ This section defines the terminology and acronyms used in this document. For
...
-
-  o  OUI: Organisation Unique Identifier. IEEE assigned prefix for EUI
-
-  o  SCHC gateway: It corresponds to the LoRaWAN Application Server. It manages
-     translation between IPv6 network and the Network Gateway (LoRaWAN Network
-     Server)
+- DevEUI: an IEEE EUI-64 identifier used to identify the device during the
+  procedure while joining the network (Join Procedure). It is assigned by the
+  manufacturer or the device owner and provisioned on the Network Gateway.
+- DevAddr: a 32-bit non-unique identifier assigned to a device wether:
```
Comment (ivajloip) [inline draft-ietf-lpwan-schc-over-lorawan.md]: What was meant by `wether`

Comment (Oliv4945) [reply draft-ietf-lpwan-schc-over-lorawan.md]: Oh thank you. I meant either :)

Comment (Oliv4945) [reply draft-ietf-lpwan-schc-over-lorawan.md]: Does de5458d solves your comment ?

**Review by jca-klk — APPROVED** [review]
Looks good. Thank you!

File: draft-ietf-lpwan-schc-over-lorawan.md (hunk 1 above)
Comment (ivajloip) [reply draft-ietf-lpwan-schc-over-lorawan.md]: Yes, thanks :slightly_smiling_face:

**Pull request merged**